virt/virt-2-6-release-notes.adoc

```diff
@@ -30,23 +30,16 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 [id="virt-2-6-new"]
 == New and changed features
 
-//CNV-7544: Microsoft's Windows Server Virtualization Validation Program (SVVP) applies to RHCOS workers
-
 * OpenShift Virtualization is certified in Microsoft’s Windows Server Virtualization Validation Program (SVVP) to run Windows Server workloads.
 +
 The SVVP Certification applies to:
 +
 ** Red Hat Enterprise Linux CoreOS workers. In the Microsoft SVVP Catalog, they are named __Red Hat OpenShift Container Platform 4 on RHEL CoreOS__.
-
 ** Intel and AMD CPUs.
 
-//CNV-9723: VMs are now migrated when a node is drained
 * Virtual machines that have the `LiveMigratable` condition set to `True` and the `spec.evictionStrategy` field set to `LiveMigrate` are now migrated when a node is drained in preparation for maintenance. You can xref:../virt/node_maintenance/virt-setting-node-maintenance.adoc#virt-setting-node-maintenance[set a node to maintenance mode in the CLI] by running the `oc adm drain` command. The `NodeMaintenance` custom resource definition is no longer supported.
```

Comment:
@ousleyp Can you please remove the last sentence from this release note? Received ack from @stu-gott to remove the line "The `NodeMaintenance` custom resource definition is no longer supported." Thanks.

Reply:
Done!